web/src/main/java/org/cbioportal/web/GenericAssayController.java

```diff
@@ -121,16 +123,16 @@
 
         List<GenericAssayData> result;
         if (interceptedGenericAssayDataMultipleStudyFilter.getMolecularProfileIds() != null) {
-            result = genericAssayService.fetchGenericAssayData(
+            result = filterEmptyGenericAssayData(genericAssayService.fetchGenericAssayData(
```

Comment:
This genericAssayService method is also used in the `StudyViewFilterApplier`. Do we want to filter out blank and na values there as well? If yes, I think you should move this filtering logic to either service layer, or, if it's simple enough, to the SQL itself. If no, I think it's probably fine to keep the logic here.

Reply:
@Luke-Sikina Currently, we don't have this need to filter out blank and na values for study view, so I'd suggest to keep this as same as molecularDataController(https://github.com/cBioPortal/cbioportal/blob/master/web/src/main/java/org/cbioportal/web/MolecularDataController.java#L119-L172)